### ISSUE TITLE
[feat] Healthcheck to consider MQTT only if  enabled

### ISSUE THREAD

**Issue by docbobo** [zwave-js/zwavejs2mqtt#469]
**Is your feature request related to a problem? Please describe.**
It's a mix between  a  bug and a feature request. Right now, documentation for the Healthcheck (/health) says that 200 OK is being returned when both MQTT and zwave are up and  running.  I am running  my  system with  MQTT  disabled  and the /health  check  returns 500. Technically,  this  is  exactly as described. However, I feel that this is not the desired  behavior.

**Describe the solution you'd like**
I'd like to see a general healtcheck like /health that takes into account  if MQTT is enabled.

**Describe alternatives you've considered**
If anybody wants to provide a general purpose kubernetes recipe, switching between different kind of probes  is not an option and will result in problems.  For  example, the deployment.yaml  in  the repo will only work if MQTT is enabled. Otherwise, it  will kill  the container after 12 attempts.

**Comment by robertsLando** [comment]
@docbobo Maybe it's not documented but you can request the healthcheck of just mqtt/zwave by adding `mqtt` or `zwave` as suffix.

Source: https://github.com/zwave-js/zwavejs2mqtt/blob/master/app.js#L225

**Comment by robertsLando** [comment]
It's documented: https://zwave-js.github.io/zwavejs2mqtt/#/guide/healthcheck?id=health-check-endpoints

**Comment by docbobo** [comment]
I am fully aware that it's documented. That's the reason why I filed this as a feature request, not as a bug. BTW, that's also what I wrote in the description above ;)

Here's why I still think it matters:
- The GitHub repository contains Kubernetes manifests. Those manifests use `/health` as a liveliness probe.
- When MQTT is enabled, that all works nicely.
- When someone disables MQTT, everything almost looks well for a while. Unless the 12 attempts are right, at which point the container is killed and restarted.

Purely from an API design perspective, I feel that there should be an API that is agnostic to all of that. An API that will report the health status of all the components that are *enabled*.

Hope this makes more sense now ;)
